README.md

```diff
@@ -29,10 +29,10 @@ The libraries in this SDK are not dependent on any operating systems. However, t
 - CMake 3.13.0 or later and a C90 compiler.
 
 - A supported operating system. The ports provided with this repo are expected to work with all recent versions of the following operating systems, although we cannot guarantee the behavior on all systems.
+	- Linux system with POSIX sockets and timer APIs. (CI tests on Ubuntu 18.04).
```

Comment:
In the prerequisites section, can we also mention that `stdint.h` is a prereq, despite not being part of C90?